README.md

```diff
@@ -1,3 +1,7 @@
 Install required roles by:
 
   ansible-galaxy install -r requirements.yml -p roles
```

Comment:
`ansible-galaxy install -r roles/requirements.yml -p roles` and then you don't need the comment in lines 4-7, and you won't have duplicate requirements.yml files